packages/Testing/PHPUnit/Behavior/MovingFilesTrait.php

```diff
@@ -49,10 +49,10 @@ protected function assertFilesWereAdded(array $expectedAddedFileWithContents): v
         foreach ($addedFilePathsWithContents as $key => $addedFilePathWithContent) {
             $expectedFilePathWithContent = $expectedAddedFileWithContents[$key];
 
-            $this->assertSame(
-                $expectedFilePathWithContent->getFilePath(),
-                $addedFilePathWithContent->getFilePath()
-            );
+            [, $expectedFilePathWithContentFilePath] = explode('_temp_fixture_easy_testing', $expectedFilePathWithContent->getFilePath());
+            [, $addedFilePathWithContentFilePath] = explode('_temp_fixture_easy_testing', $addedFilePathWithContent->getFilePath());
+
+            $this->assertSame($expectedFilePathWithContentFilePath, $addedFilePathWithContentFilePath);
```

Comment:
Looks good :+1: 

Could you add a comment why it's done this way? Just so in the future we know what is going on here :)

Reply:
comment added https://github.com/rectorphp/rector-src/pull/129/commits/f17add75b6acb47693c04439bd7f150e2e60c10d

Reply:
isn't the fix in https://github.com/rectorphp/rector-src/pull/120 for the same problem way more elegant?

Reply:
I think this is not about line-endings, nor slahes in paths, but different rules for `sys_get_temp_dir()` functions

Reply:
still the changes in #120 fixed the problem at hand in 
![grafik](https://user-images.githubusercontent.com/120441/120196671-6558ea00-c220-11eb-8768-23f6bfb43231.png)

Reply:
@staabm Both should be included :+1: